server/svix-server/src/queue/redis.rs

```diff
@@ -23,6 +23,10 @@
 //! processed now, and the currently processing queue for tasks that have timed out and should be
 //! put back on the main queue.
 
+// This lint warns on `let _: () = ...` which is used throughout this file for Redis commands which
+// have generic return types. This is cleaner than the turbofish operator in my opinion.
+#![allow(clippy::let_unit_value)]
```

Comment:
Why not have it global to all of our project?

Reply:
I generally agree with the lint when the turbofish isn't involved, ie. if there is a function that specifically doesn't return anything, it's cleaner to just `call_it()` instead of `let _: () = call_it()`. But it's only in this file where the warning has occurred because of generic return types on Redis commands.

If this pops up elsewhere I'd be okay with making it apply to the whole crate.